scripts/lc-builds/toss4_icpx.sh

```diff
@@ -57,3 +57,17 @@ cmake \
   "$@" \
   ..
 
+echo
+echo "***********************************************************************"
+echo
+echo "cd into directory build_${BUILD_SUFFIX} and run make to build RAJA"
+echo
+if [[ ${COMP_VER} == 2024.2.1 ]]
```

Comment:
Why do we have this comment in here that is tied to a specific version of the compiler?

Reply:
I don't know, but the old logic wasn't printing any of this at all.